std_msgs/String command 
---
ml4kp_bridge/Trajectory trajectory
